src/bench/bench.cpp

```diff
@@ -61,6 +59,12 @@ void benchmark::BenchRunner::RunAll(const Args& args)
 
         Bench bench;
         bench.name(p.first);
+        if (args.min_time > 0ms) {
+            // convert to nanos before dividing to reduce rounding errors
+            std::chrono::nanoseconds min_time_ns = args.min_time;
```

Comment:
b8f16c3

- per `doc/developer-notes.md`, it looks like `#include <chrono>` should be added to bench.cpp and bench_bitcoin.cpp:
```
- Every `.cpp` and `.h` file should `#include` every header file it directly uses classes, functions or other
  definitions from, even if those headers are already included indirectly through other headers.

  - *Rationale*: Excluding headers because they are already indirectly included results in compilation
    failures when those indirect dependencies change. Furthermore, it obscures what the real code
    dependencies are.
```

- pico-nit, feel free to ignore
```suggestion
            const std::chrono::nanoseconds min_time_ns{args.min_time};
```

Reply:
hm, that include guideline is a bit hard to follow. There are lots of other missing includes, e.g. `bench_bitcoin.cpp` would need at least chrono, cstdint, vector, sstream, string, iostream. Should I add these as well?

About the constructor, I think that is one of the cases where assignment is better: chrono's assignment is more strict than the constructor, you can't do e.g. `std::chrono::nanoseconds x = 123` but you can do `std::chrono::nanoseconds x{123}`. So with the assignment it's certain that `args.min_time` is actually another chrono type that convertible without loss of information, and not just any number that could represent anything.

Reply:
I've now added the headers in clean up includes commit